content/internships/contents.lr

```diff
@@ -12,24 +12,10 @@ Creative Commons is currently participating in the current rounds of the followi
 
 ### Google Season of Docs
 
-- [Program website](https://developers.google.com/season-of-docs/)
-- CC's [Project Ideas](/internships/project-ideas/)
-
-If you're interested in this program, please review the resources below and come talk to us on Slack in `#cc-gsoc-outreachy`.
-
-### Google Summer of Code 
```

Comment:
GSoC doesn't end until the end of the month, only Outreachy has ended.

Reply:
Noted, my mistake. I falsely assumed that they had identical timelines.

Reply:
Resolved in 99fe0d3.